packages/gherkindoc/test/FormatSupport.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Formatting support
```

Comment:
@jayden-chan I tried to write these as real self-describing behaviours. You could add some more here, as a form of documentation